index.js

```diff
@@ -554,3 +561,14 @@ function destroyStream (stream) {
     else if (typeof stream.emit === 'function') stream.emit('close')
   }
 }
+
+function normalizeGlobalLabels (labels) {
+  const result = {}
+  if (!labels) return result
+
+  for (let key of Object.keys(labels)) {
+    result[key] = labels[key].toString()
```

Comment:
We should verify if we're allowed to support numbers and booleans, and not only strings

Reply:
From 6.7 APM server accepts it - https://github.com/elastic/apm-server/pull/1712, But we are doing the same `toString` because we support 6.5+ versions.

Reply:
In that case, I would consider it a breaking change to make this support numbers and booleans, so we should do that in another PR targeted at landing in the next major.

Reply:
Actually, it would just get fully ignored on pre-6.7 versions because global labels are on metadata messages only, right? So it actually _wouldn't_ be a backwards-compatibility concern to send the data up with numbers and booleans. 🤔

Reply:
I've changed it to allow number and boolean types.